tests/http/http.go

```diff
@@ -33,3 +37,57 @@ func perform(req *http.Request, r http.Handler, headers http.Header) *httptest.R
 	r.ServeHTTP(response, req)
 	return response
 }
+
+func ReadResponseBody(respBody interface{}) []byte {
+	var bodyByte []byte
+	var err error
+	byteBufferBody, isByteBuffer := respBody.(*bytes.Buffer)
+	readCloserBody, isReadCloser := respBody.(io.ReadCloser)
+
+	if isByteBuffer {
+		bodyByte, err = ioutil.ReadAll(byteBufferBody)
+	} else if isReadCloser {
+		bodyByte, err = ioutil.ReadAll(readCloserBody)
+	}
+
+	if err != nil {
+		log.Error(errorconf.ReadResponseBodyFailure, err)
```

Comment:
@gutakk Does `log.Error` fail the test? If no, then do you think we should make the test failed?

Reply:
Good catch!. `log.Error` won't fail the test.
Updated to use fatal instead in c087a2d570a3b6a6e604195f15866d1806453119